archive/cap-of-vector-db.md

```diff
@@ -0,0 +1,30 @@
+
+# What is CAP?
+- [CAP Theorem on Wikipedia](https://en.wikipedia.org/wiki/CAP_theorem)
+
+# Explanation of CAP
```

Comment:
The main point of CAP theorem is that we cannot construct a distributed system satisfying all the good properites called CAP. How about adding this to the content? Without this content, the reader may be confused with why all the conditions are satisfied among the products.

Reply:
I'll add to content.